docs/appendix/changelog.md

```diff
@@ -10,6 +10,242 @@ aliases:
+
+This update enables you to get a closer examination of your image and streamline your image annotation experience by making it easier to add bounding boxes and labels in the **Data** tab.
+
+With the latest improvements, you can now:
+
+- Navigate between images using the arrow keys in the main image view
+- Expand images for a more detailed inspection by clicking the expand button on the right image panel
+- Move between full-screen images effortlessly with the <> arrow buttons or arrow keys
+- Return to the standard view by using the escape key or collapse button
+
+### Added: Duplicate component
+
+You now have the ability to duplicate any config component, service, module, remote, or process.
+This functionality enables the duplication of any configured component and simplifies the process of creating identical configurations for resources.
+
+To use this feature:
+
+- Click on the duplicate component icon at the top right of any resource
+- Optionally, you can modify the component name to distinguish it
+- Adjust any attributes, such as motor pin numbers
+
+## July 2023
+
+### Added: Apple SSO Authentication
+
+Viam now supports sign-up/log-in through Apple Single Sign-On.
+
+Note that currently, accounts from different SSO providers are treated separately, with no account merging functionality.
+
+### Added: Viam VS Code Extension
+
+There is now a Visual Studio Code extension for Viam that offers a set of code snippets, streamlining the process of building robot applications by providing predefined code for common tasks.
+
+### Improved: Arm component API
+
+You can now access a [`GetKinematics` method](https://github.com/viamrobotics/api/blob/main/proto/viam/common/v1/common.proto#L186-L203) in the arm API, allowing you to request and receive kinematic information.
+
+With this update:
+
+- Request kinematic information through the arm client
+- Receive kinematic information from arm implementations (modular arms)
+- Experience enhanced functionality through added tests
+
+## June 2023
+
+### Added: View sensor data within Viam
+
+You can now [view your sensor data](https://app.viam.com/data/view?view=sensors) directly in the Viam app to verify data creation and accuracy.
+If you depend on sensor data to plan and control robot operations, this feature increases access to data and supports a more efficient workflow.
+
+### Added: Session management in the Python SDK
+
+The Python SDK now includes sessions, a safety feature that automatically cancels operations if the client loses connection to your robot.
+
+Session management helps you to ensure safer operation of your robot when dealing with actuating controls.
+Sessions are enabled by default, with the option to disable them by setting `RobotClient.Options.disable_sessions = True`.
+
+### Added: Connect an ODrive motor controller as a Viam module
+
+You can integrate and control ODrive motor controllers with Viam using a Python modular resource.
```

Comment:
I don't think it really matters which language it is in since you can use it with either SDK, so adding the language may cause unnecessary confusion
```suggestion
You can integrate and control ODrive motor controllers with Viam using a modular resource.
```